modules/microshift-olm-deploy-ops-spec-ns.adoc

```diff
@@ -94,7 +94,7 @@ spec: <1>
 ----
 $ oc apply -f _<og.yaml>_ <1>
 ----
-<1> Replace _<og.yaml>_ with the name of your operator group configuration file.
+<1> Replace `<og.yaml>` with the name of your operator group configuration file.
```

Comment:
```suggestion
<1> Replace `<og>.yaml` with the name of your operator group configuration file.
```

Reply:
changes are made in both lines 95 and 97. Is that fine?